test/e2e/configurer/chain/commands_btcstaking.go

```diff
@@ -188,7 +196,15 @@ func (n *NodeConfig) CommitPubRandList(fpBTCPK *bbn.BIP340PubKey, startHeight ui
 	// gas
 	cmd = append(cmd, "--gas=500000")
 
-	_, _, err := n.containerManager.ExecTxCmd(n.t, n.chainId, n.Name, cmd)
+	allTxArgs := []string{fmt.Sprintf("--chain-id=%s", n.chainId), "--gas-prices=0.002ubbn", "-b=sync", "--yes", "--keyring-backend=test", "--log_format=json", "--home=/home/babylon/babylondata"}
+	txCommand := append(cmd, allTxArgs...)
+	return n.containerManager.ExecCmd(n.t, n.Name, txCommand, success)
+
+	// return n.containerManager.ExecTxCmd(n.t, n.chainId, n.Name, cmd)
```

Comment:
some leftover ?

Reply:
e2e is not going nice, still need to fix the e2e test check before merge

Reply:
finally https://github.com/babylonlabs-io/babylon/pull/204/commits/a5bc7f3bf7827865a4e93f71ded856e128b9026e